pages/doc/integrations_tas_howto.md

```diff
@@ -346,6 +346,63 @@ Log in to your Wavefront instance (for example, `https://example.wavefront.com`)
 3. Explore one or two dashboards. [Examine Data with Dashboards and Charts](ui_examine_data.html) has an overview and includes a video.
 4. As appropriate, clone any of the existing dashboards to add charts, modify queries, and more. See [Create, Customize, and Optimize Dashboards](ui_dashboards.html) and [Create and Customize Charts](ui_charts.html).
 
+### Optional 1: Use the service broker to send custom application metrics to the wavefront proxy
+
+1. In Ops Manager, in the "Wavefront Proxy Config" panel,
+   check the "Enable service broker bindings" checkbox.
+2. To provision a wavefront-proxy service instance:
+   `cf create-service wavefront-proxy standard SERVICE_INSTANCE_NAME`.
+3. To bind apps to the service instance: `cf bind-service APP_NAME SERVICE_INSTANCE_NAME`.
+   If you already have an existing app and want to update its service instance reference from the v3 broker, first unbind it with this command:
+   `cf unbind-service APP_NAME SERVICE_INSTANCE_NAME`.
+
+Notes:
+1. For information about how apps can send metrics to the Wavefront proxy, see the [cloud-foundry-sampleapp](https://github.com/wavefrontHQ/cloud-foundry-sampleapp).
+
+### Optional 2: Connect multiple proxy instances to an IAAS load balancer
+
+#### Set up the load balancer in your IAAS
+If using multiple Wavefront VMs,
+you will want to use your IAAS of choice to set up a load balancer.
+BOSH DNS does not provide load balancing,
+so just increasing the VM count by itself will result in one VM taking all the requests.
+
+An in depth guide on configuring load balancers in different IAASes is beyond the scope of this document.
+You can see a detailed example across two sections of the TAS for VMs documentation:
+1. Initial setup of load balancers is required in
+   [AWS](https://docs.pivotal.io/ops-manager/3-0/aws/prepare-env-manual.html) and
+   [GCP](https://docs.pivotal.io/ops-manager/3-0/gcp/prepare-env-manual.html#loadbalancer),
+   but not for Azure.
+2. After initial setup,
+   [this](https://docs.pivotal.io/application-service/3-0/operating/configure-lb.html)
+   document can be referenced for finalizing the load balancers in all IAAS.
+
+You will want to adjust the naming in these docs to suit a wavefront proxy.
+The portions of these docs that detail interactions with Ops Manager can be ignored,
+since we will talk about that in the next section.
+For GCP, record the backend service name of your load balancer,
+for AWS, you will want the actual Elastic Load Balancer name,
+and for Azure, you will want the Azure Load Balancer name.
+
+#### Configure the tile to use the load balancer for the Wavefront Proxy
+1. Enter the `Resource Configuration` tab for that tas2to tile in your Opsman config.
+2. Click on `Wavefront Proxy Configuration` in the left column.
+3. Enter `http:${load_balancer_name}`,
```

Comment:
3. Enter `http:${load_balancer_name}`, where `load_balancer_name` is the load balancer or backend service name from the initial setup.